docs/content/guides/3.performance/2.core-web-vitals.md

```diff
@@ -0,0 +1,174 @@
+
+
+## Largest Contentful Paint (LCP)
+
+Largest Contentful Paint (LCP) is the time it takes for the browser to paint the largest contentful element on the page. To improve the LCP metric, you need to start loading that element as fast as possible. 
+
+If it needs an asset like JavaScript, CSS, image, or font, use the "preloading". It's a technique for telling a browser to download a resource before it discovers it's needed.
+
+For example, you might have a font that the browser can discover late (because it first needs to download and parse the CSS file), but you know that it's critical to your website. You can preload it, so once the browser parses the CSS and finds out that the font is needed, it will already have it downloaded. 
+
+[Read more about preloading critical assets](https://web.dev/preload-critical-assets/).
+
+### Preloading Images
+
+If there is an image in the viewport, preloading it will help the browser load it as soon as possible. 
+
+Frameworks like [Next.js](https://nextjs.org/docs/pages/api-reference/components/image#priority) and [Nuxt](https://image.nuxt.com/usage/nuxt-img#preload) have image libraries that provide preloading functionality, but it is also possible to preload images manually using the `<link rel="preload">` tag.
+
+::code-group
+```html[Next.js]
+https://nextjs.org/docs/pages/api-reference/components/image#priority
+<Image
+  src="/image.png"
+  priority={true}
+/>
+```
+```html[Nuxt]
+https://image.nuxt.com/usage/nuxt-img#preload
+<NuxtImg 
+  src="/image.png" 
+  preload 
+/>
+```
+```html[Other]
+https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/prefetch
+<head>
+  <link rel="prefetch" href="/image.png" fetchpriority="high" as="image" />
+</head>
+```
+::
+
+### Making Resources Smaller and Faster
+
+If you are already preloading resources, you can also improve LCP by making the resources load faster. This will help the browser load and paint the content faster.
+
+A few examples of this are:
+
+- Using the proper size image and not unnecessarily using a larger image
+- Using modern image formats
+- Reducing the web font size
+- Using a CDN to reduce the request times
+
+### Identifying the Largest Contentful Paint
+
+The easiest way to learn which element is the Largest Contentful Paint is using [PageSpeed Insights](https://pagespeed.web.dev/) page. After generating the report, the LCP metric and what element it relies on is displayed under the "Largest Contentful Paint element" diagnostic.
+
+Another option is to run Lighthouse in Chrome DevTools, described on [Google Developers](https://developers.google.com/web/tools/lighthouse#devtools) website.
+
+**See More**:
+- [Google's Guide on Optimizing LCP](https://web.dev/articles/optimize-lcp)
```

Comment:
suggestion: I would keep it as a part of `## Largest Contentful Paint (LCP)` header instead making it separate heading.

The reason for that is that in the next subheadings you are showing examples how to improve the vital score